build/Publish/PublishNupkgToTransportFeed.targets

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
+
+  <PropertyGroup>
+    <!-- Capture 'OutputPath' before Publish.csproj modifies it -->
+    <OutputPathTransportFeed>$(OutputPath)</OutputPathTransportFeed>
+    <BuildTasksFeedDllVersion>1.0.0-prerelease-01929-02</BuildTasksFeedDllVersion>
+    <BuildTasksFeedDll>$(NuGet_Packages)\Microsoft.DotNet.Build.Tasks.Feed\$(BuildTasksFeedDllVersion)\lib\desktop\Microsoft.DotNet.Build.Tasks.Feed.dll</BuildTasksFeedDll>
+  </PropertyGroup>
+
+  <UsingTask TaskName="PushToBlobFeed" AssemblyFile="$(BuildTasksFeedDll)" />
+
+  <ItemGroup>
+    <NupkgsForPublishing Include="$(OutputPathTransportFeed)Packages\Microsoft.NET.Sdk.*.nupkg" />
+    <NupkgsForPublishing Include="$(OutputPathTransportFeed)Packages\Microsoft.NET.Build.Extensions.*.nupkg" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <RelativePath>packages</RelativePath>
+    <TransportFeedContainerName>$(TRANSPORTFEED_STORAGE_CONTAINER)</TransportFeedContainerName>
+    <TransportFeedContainerName Condition="'$(TransportFeedContainerName)' == ''">dotnet-core</TransportFeedContainerName>
+    <TransportFeedCloudDropAccessToken>$(TRANSPORTFEED_STORAGE_KEY)</TransportFeedCloudDropAccessToken>
+    <TransportFeedCloudDropAccountName>$(TRANSPORTFEED_STORAGE_ACCOUNT)</TransportFeedCloudDropAccountName>
+    <TransportFeedCloudDropAccountName Condition="'$(TransportFeedCloudDropAccountName)' == ''">dotnetfeed</TransportFeedCloudDropAccountName>
+  </PropertyGroup>
+
+  <Target Name="PublishNupkgToTransportFeed"
+          Condition=" '$(PUBLISH_NUPKG_TO_TRANSPORT_FEED)' == 'true' "
+          DependsOnTargets="RestorePackageForTransportFeed;
+                            PushNupkgToTransportFeed" />
+
+  <Target Name="RestorePackageForTransportFeed">
+    <PropertyGroup>
+      <NewLineTF>
+        <![CDATA[
+]]>
+      </NewLineTF>
+      <SetNuget_PackagesTF>set NUGET_PACKAGES=$(NuGet_Packages)</SetNuget_PackagesTF>
+      <SetDotnet_Skip_FirstTime>set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true</SetDotnet_Skip_FirstTime>
+      <PublishProjectFile>"$(MSBuildThisFileDirectory)Publish.csproj"</PublishProjectFile>
+    </PropertyGroup>
+
+    <Exec Command="$(SetNuget_PackagesTF)$(NewLineTF)$(SetDotnet_Skip_FirstTime)$(NewLineTF)$(DotNetTool) restore $(PublishProjectFile) /v:minimal"
+          WorkingDirectory="$(RepositoryRootDirectory)" />
+  </Target>
+
+  <Target Name="PushNupkgToTransportFeed" >
```

Comment:
This target appears to be re-implementing some of the functionality of Publish.targets from the package.  Is this a temporary step (subsequent versions of the package have changed considerably)?

Reply:
This is a temporary step in the sense that the SDK repo will be folding in the CLI repo. The PR is to support step 1 completion.

Reply:
I'd be worried about temporary steps because they often become servicing branches and then updating them can be painful because they vary so much from other branches.  ie, I would hate for the servicing story to be, rewrite this target to match the new target in the package.  I understand that the api for this package was rough for the version you're currently using though.